firefox-ios/Client/Assets/Images.xcassets/chevronDownTiny.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "chevronDownTiny.svg",
```

Comment:
SVGs should not be added to the project. @cwzilla could we add a pdf of this image to the repo?

Reply:
Didn't know this. I thought that as long as the icons are in the acorn repo, they are good to be added. Is there a document somewhere, or should we add this to our guidelines?

Reply:
There is documentation in the [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/Strings,-%60AccessibilityIdentifiers%60,-%60ImageIdentifiers%60#standardimageidentifiers). 
If we don't use PDFs our automatic image update script won't work anymore.

Reply:
Also something I learned today!